src/NovelRT.Interop/Graphics/NrtBasicFillRect.cpp

```diff
@@ -126,7 +126,7 @@ extern "C"
         }
 
         BasicFillRect* cppRect = reinterpret_cast<BasicFillRect*>(rect);
-        cppRect->setColourConfig(*reinterpret_cast<RGBAConfig*>(&inputColourConfig));
+        cppRect->setColourConfig(*reinterpret_cast<RGBAConfig*>(inputColourConfig));
```

Comment:
Is it only on this rect type this is a problem?

Reply:
I checked [`TextRect`](https://github.com/novelrt/NovelRT/blob/main/src/NovelRT.Interop/Graphics/NrtTextRect.cpp#L137) and [`ImageRect`](https://github.com/novelrt/NovelRT/blob/main/src/NovelRT.Interop/Graphics/NrtImageRect.cpp#L163) and they were fine.